packages/suite/src/views/onboarding/steps/Final/index.tsx

```diff
@@ -201,7 +175,6 @@ const FinalStep = () => {
                                     <Translation id="TR_DEVICE_SETTINGS_DEVICE_EDIT_LABEL" />
                                 </OptionText>
                             </Option>
-                            <Divider />
```

Comment:
Because it was just a `div` with left or top margin based on css media query. I replaced it with `gap` css property which should do the same thing and allows to decouple the styled components. Until now it was required that the same media query in two components a) change the row/column layout and b) change the direction of the margin because of that.